**Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек.**  
**a) Какова вероятность, что ни одна из них не перегорит в первый день?**  
**б) Какова вероятность, что перегорят ровно две?**

Поскольку в задаче вероятность наступления события мала, а количество испытаний велико, для решения задачи воспользуемся формулой **Пуассона**:  

$P_m=\frac{\lambda^m}{m!}\cdot e^{-\lambda}$, где $\lambda=p\cdot n$

Создадим функцию расчета по формуле Пуассона:

In [4]:
from math import factorial, exp
def poisson_distr(m,p,n):
    lamb=p*n
    return exp(-lamb)*(lamb**m)/factorial(m)

а) по первому условию задания -  

                                    p=0.0004 - вероятность перегорания лампочки в первый день,  
                                    n=5000 - количество ламп,  
                                    m=0 - событие не наступит, т.е. ни одна лампа не сгорит  
Вычислим:

In [5]:
print(f'P = {poisson_distr(0,0.0004,5000): .4f}')

P =  0.1353


___  
_Ответ: вероятность того, что ни одна из ламп не перегорит в первый день составляет **P=0.1353** или **$\approx$ 13.53%**_  

___

б) по второму условию задания -  

                                    p=0.0004 - вероятность перегорания лампочки в первый день,  
                                    n=5000 - количество ламп,  
                                    m=2 - в первый день должны перегореть две лампы  
Вычислим:

In [6]:
print(f'P = {poisson_distr(2,0.0004,5000): .4f}')

P =  0.2707


___  
_Ответ: вероятность того, что в первый день перегорят 2 лампы составляет **P=0.2707** или **$\approx$ 27.07%**_  

___